code/missionui/missionscreencommon.cpp

```diff
@@ -1939,4 +1925,22 @@ void draw_model_rotating(model_render_params *render_info, int model_id, int x1,
 	}
 }
 
+/**
+ * @brief add and rotate lights for all the non-gameplay ship rendering instances
+ */
+void common_setup_room_lights()
+{
+	light_reset();
+	auto tempv = vm_vec_new(-1.0f,0.3f,-1.0f);
+	auto tempc = hdr_color(1.0f,0.95f,0.9f, 0.0f, 1.5f);//use 3 for qaz shaders
+	light_add_directional(&tempv,&tempc);
+	tempv.xyz={-0.4f,0.4f,1.1f};
+	tempc = hdr_color(0.788f,0.886f,1.0f,0.0f,1.5f);//use 3 for qaz shaders
+	light_add_directional(&tempv,&tempc);
+	tempv.xyz={0.4f,0.1f,0.4f};
+	tempc = hdr_color(1.0f,1.0f,1.0f,0.0f,0.4f);//use 0.8 for qaz shaders
+	light_add_directional(&tempv,&tempc);
```

Comment:
Eventually, these should be configurable from lighting_profiles I imagine, right?

Reply:
Yes, that's on my to-do list. Probably broken out by context if desired.